### DISCUSSION TITLE
windows 如何裸核运行 mihomo，并且开启使用 Metacubexd web控制面板 ？

### DISCUSSION THREAD

**Discussion by Mitchell08** [MetaCubeX/mihomo#1682] [Q&A]
windows 如何裸核运行 mihomo，并且开启使用 Metacubexd web控制面板 ？

congfig.yaml 用的是 wiki 里的快捷配置，但是好像运行不起来，求指点啊，非常感谢！

![屏幕截图 2024-11-27 173933](https://github.com/user-attachments/assets/d1a948c9-e87e-42ed-9e05-248b5287fd17)

![屏幕截图 2024-11-27 173822](https://github.com/user-attachments/assets/55929170-5368-4605-afc0-4b8e7e98d2d0)

![屏幕截图 2024-11-27 173811](https://github.com/user-attachments/assets/98503a5c-062b-48c3-b9ef-1474fbc063fa)

![屏幕截图 2024-11-27 173800](https://github.com/user-attachments/assets/5b8335d5-fc29-416a-a09e-8d0c58e40096)

![屏幕截图 2024-11-27 174217](https://github.com/user-attachments/assets/10d703bb-6222-4448-8c91-4cb1e55ca84b)

**Comment by hgoev** [comment]
可以参考这个https://linux.do/t/topic/236719
但是我把裸核跑起来了，也进入了控制面板，但是没办法代理，不知道为什么。。。。

**Reply by sm0012333** [reply]
> 可以参考这个https://linux.do/t/topic/236719 但是我把裸核跑起来了，也进入了控制面板，但是没办法代理，不知道为什么。。。。

已搞定。

**Comment by sm0012333** [comment]
目前我只会用zashboard面板

**Comment by iKira** [comment]
用这个：
https://github.com/winsw/winsw
